### DISCUSSION TITLE
Running Avante project as a Docker Container #17189

### DISCUSSION THREAD

**Discussion by MyCodeBits** [yetone/avante.nvim#454] [Q&A]
My requirement to have Avante IDE open in the browser (thus Avante exposed on a port).

- One of the option is that if Avante project is available as a Docker image. Is there a known way to run Avante as a Docker container?
- Any other option by which it can be exposed on a port and accessible as locahost:[PORT] ?

Thanks.

**Comment by aarnphm** [comment]
This is unrelated. avante.nvim is just a neovim plugin, it did not run any models whatsoever 😄 

If you want to run neovim in a docker container there are plenty of ways to do that, just google.

Closed due to unrelated.

**Comment by MyCodeBits** [comment]
@aarnphm 
Thanks for the response and correcting me to focus on Neovim.

>If you want to run neovim in a docker container there are plenty of ways to do that, just google.

I did search before posting it. I do see articles that can get GUI applications run in Docker container and have Ui accessible. But they fall short of talking about how will the GUI be exposed on a PORT.

Would be great if u can share any helpful link for it.
